cmd/embedded-cluster/install.go

```diff
@@ -265,7 +265,7 @@ func runOutro(c *cli.Context) error {
 // file is created. Resulting kubeconfig is stored in the configuration dir.
 var installCommand = &cli.Command{
 	Name:  "install",
-	Usage: "Installs and starts a new cluster",
+	Usage: "Install a cluster",
```

Comment:
This is more than a cluster to the end user. In fact, they think of it as the app more so than the cluster. WDYT about "Install \<binary name>"? Maybe "Install \<binary name> cluster", but in a single-node situation, the word cluster feels confusing.

This lines up with the `Install and manage %s", name` wording you used below too.

Reply:
OK, `Install <binary name>` it is.

Reply:
```suggestion
	Usage: fmt.Sprintf("Install %s", defaults.BinaryName()),
```